mino/router/tree/example_test.go

```diff
@@ -0,0 +1,65 @@
+package tree
+
+import (
+	"fmt"
+
+	"go.dedis.ch/dela/mino"
+	"go.dedis.ch/dela/mino/minogrpc/session"
+)
+
+func ExampleRouter_New() {
+	router := NewRouter(session.AddressFactory{})
+
+	addrA := session.NewAddress("127.0.0.1:2000")
+	addrB := session.NewAddress("127.0.0.1:3000")
+
+	players := mino.NewAddresses(addrA, addrB)
+
+	table, err := router.New(players)
+	if err != nil {
+		panic("routing table failed: " + err.Error())
+	}
+
+	routes, voids := table.Forward(table.Make(addrA, []mino.Address{addrB}, []byte{}))
+	fmt.Println(voids)
+
+	for to := range routes {
+		fmt.Println(to)
+	}
+
+	// Output: map[]
+	// 127.0.0.1:3000
+}
+
+func ExampleTable_Prelude() {
+	routerA := NewRouter(session.AddressFactory{})
+
+	addrA := session.NewAddress("127.0.0.1:2000")
+	addrB := session.NewAddress("127.0.0.1:3000")
+
+	players := mino.NewAddresses(addrA, addrB)
+
+	table, err := routerA.New(players)
+	if err != nil {
+		panic("routing table failed: " + err.Error())
+	}
+
+	handshake := table.Prelude(addrB)
+
+	// Send the handshake to the address B..
+
+	routerB := NewRouter(session.AddressFactory{})
+
+	tableB, err := routerB.TableOf(handshake)
```

Comment:
Similarly out of scope:

`TableOf` has a similar issue.
What we're doing here (if I understand correctly) is more like a `GenerateTableFrom` no?

Reply:
Yes, I can change to this.